jinja_utils.py

```diff
@@ -104,7 +134,15 @@ def parse_string(string, params, autoescape=True):
 
 
 def evaluate_object(obj, params):
-    """Returns a copy of `obj` after parsing strings in it using `params`."""
+    """Returns a copy of `obj` after parsing strings in it using `params`.
+
+    Args:
+        obj: *. An object to parse strings in it.
+        params: list(*). The parameters to parse the string with.
```

Comment:
How do you know it's a list?

Reply:
Yes, print it please. All that line tells you is that params is an iterable but it doesn't tell you what kind of iterable it is.

Reply:
Done. The type should be `dict(str, *)`. It's verified from test file [here](https://github.com/oppia/oppia/blob/0804599fea5464935d96ffdbc9feef726cfea695/jinja_utils_test.py#L46).